scripts/eda.py

```diff
@@ -62,7 +62,7 @@ def main(processed_training_data, plot_to):
         table.set_fontsize(10)
         table.auto_set_column_width(col=list(range(len(freq_percent_df.columns))))
 
-        file_path = os.path.join(plot_to, "figures", f"{column}_frequency_table.png")
+        file_path = os.path.join(plot_to, "tables", f"{column}_frequency_table.png")
```

Comment:
As mentioned in issue #25, if we save them as tables, there's not gonna be color gradients/highlights in the tables, would that be ok for our final report?

Reply:
Oh! We can save them as png files, I just thought we could move them to the tables directory for organization. Is that okay?

Reply:
Yes, I guess it does not matter where we put them as long as we can refer to the right directory in the `qmd`.